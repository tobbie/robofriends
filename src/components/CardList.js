import React from 'react'
import Card from './Card'



const CardList = ({robots}) => {
  const cardsArray =   robots.map((user, index) => {
                            return ( <Card key= {index} name = {user.name} email = {user.email} id = {user.id}/>)
                        });
                        console.log('CardList')
   return(<div>
            {cardsArray}
         </div>);
}

export default CardList;