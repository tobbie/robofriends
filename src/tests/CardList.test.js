import {shallow} from 'enzyme'
import React from 'react'
import CardList from '../components/CardList'

describe('cardlist tests',()=> {
    
    const mockRobots = [
        {
            id : 1,
            name : 'Sam Jones',
            email : 'samjones@example.com'
        }
    ]
    it('expect to render cardlist component',() => {
        expect(shallow(<CardList robots = {mockRobots}/>)).toMatchSnapshot();
        
    });
})