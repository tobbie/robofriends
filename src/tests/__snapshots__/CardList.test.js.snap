// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cardlist tests expect to render cardlist component 1`] = `ShallowWrapper {}`;
